cypress/support/commands.js

```diff
@@ -32,10 +32,59 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
+Cypress.Commands.add('getByDataQa', (selector) => {
+  cy.get(`[data-qa="${selector}"]`);
+});
+
 Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
   cy.request('POST', '/users', {
     email,
     username,
     password
   });
 });
+
+Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+  cy.request('POST', '/users', {
+    user: {
+      email,
+      username,
+      password
+    }
+  }).then(response => {
+    const user = {
+      bio: response.body.user.bio,
+      effectiveImage: "https://static.productionready.io/images/smiley-cyrus.jpg",
+      email: response.body.user.email,
+      image: response.body.user.image,
+      token: response.body.user.token,
+      username: response.body.user.username,
+    };
+    window.localStorage.setItem('user', JSON.stringify(user));
+    cy.setCookie('auth', response.body.user.token);
+  });
+});
```

Comment:
In the `login` command, the `username` parameter is not used in the request body. Consider removing it from the parameters if it's not needed for the login process.